### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by Walkman100**
Used a spelling checker to fix spelling.
- No commands or functionality has changed
- Just say if I should remove some changes, squash into one commit e.t.c.

### CODE REVIEW

File: bot/bot.lua [hunk 1]
```diff
@@ -41,7 +41,7 @@ function on_binlog_replay_end()
 end
 
 function msg_valid(msg)
-  -- Dont process outgoing messages
+  -- Don't process outgoing messages
```
Comment (Walkman100) [inline bot/bot.lua]: Not sure if apostrophes `'` need to be escaped, since I don't know how Lua comments work:
```
--[[
<code>
]]--
```
Is all I know.

Comment (rockneurotiko) [inline bot/bot.lua]: No, they don't need to be escaped :)

**Comment by rockneurotiko** [comment]
Thanks!

Please, squash in one commit :)

**Comment by Walkman100** [comment]
Ok :D

**Comment by Walkman100** [comment]
Done :+1:

**Pull request merged**